### DISCUSSION TITLE
Printing stdout to a msgbox

### DISCUSSION THREAD

**Discussion by newdayrising** [AllenDang/giu#198] [Q&A]
Hello All!

I'm sort of a Go newb and can't quite figure out how to print all of Stdout to a msgbox while my Go program is running. Can someone point me in the right direction?

Thank you!

**Comment by AllenDang** [comment]
@newdayrising Can you specify more clear about what are you trying to do?

**Reply by gucio321** [reply]
@newdayrising do you mean redirrecting all the output from `log` package into some variable?
if so, just create your own `io.Writer` implementation pointing to `os.Stdout` AND your own variable.
than simply assign instance of your writer to log package's output by calling `log.SetOutput`.
